autoscheduler/frontend/src/components/LandingPage/About/About.tsx

```diff
@@ -0,0 +1,151 @@
+import * as React from 'react';
+import {
+  Dialog, DialogContent, DialogTitle, IconButton, Link, ThemeProvider, Typography,
+} from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+import { GitHub, LinkedIn } from '@material-ui/icons';
+import { Contributor, contributors, pastContributors } from '../../../types/Contributors';
+import * as styles from './About.css';
+import { whiteButtonTheme } from '../../../theme';
+
+const About: React.FC = () => {
+  const [open, setOpen] = React.useState(false);
+
+  /**
+   * Creates a row to be used in the contributor table
+   * @param contributor the Contributor to make the row for
+   */
+  function getContributorRow(contributor: Contributor): JSX.Element {
```

Comment:
Since this doesn't rely on state, move it outside the component so the function isn't recalculated on each rerender

Reply:
Mind elaborating on how this would make it not recalculate on rerender? I would think it would still since `contributorsJSX` calls it anyways

Reply:
I meant the function itself will be recalculated since functions inside react components don't have stable identities unless you wrap them in a `useCallback`